CONTRIBUTING.md

```diff
@@ -12,6 +12,9 @@ Zapraszamy wszystkich do współpracy nad projektem:
 Jest mnóstwo sposobów w jaki możesz nam pomóc:
 * Rób razem z nami audyt kodu.
 * Dodawaj swoje `Pull Request`y.
+  * Obecnie jednym z wymagań jest, żeby commity w `PR` były podpisane kluczem który jest dodany do Github-a
```

Comment:
CZy jest procedura weryfikująca takie podpisy? Kiedyś prowadziłem projekt w którym musieliśmy przy tworzeniu nowego binarnego release ze źródeł które były zweryfikowane podpisami - i mieliśmy listę takich publicznych kluczy które są "OK' - i weryfikowaliśmy te klucze przy releasie. 

Czy taki proces i lista kluczy które są "zaufane" istnieje ?

Reply:
@potiuk to dobre pytanie. Dowiedziałem się o tym wymaganiu przez email-a i zadałem dokładnie takie samo pytanie. Jak na razie nie dostałem odpowiedzi.

W odpowiedzi jaką zostawiłem napisałem, że idealnie by było, żeby powstał pęk kluczy autoryzowanych które wysyłane są do prowadzących projekt. Żeby zapewnić, że klucze są wiarygodne, powinny być podpisane przez np. profil zaufany danej osoby która podpisała CLA.
W takim wypadku na koniec ma się listę poprawnych kluczy + potrzebny jest skrypt który by weryfikował podpisy commitów do tego istniejącego pęku kluczy. 

Jak na razie nie znam szczegółów, wiem, jednak, że same commity są oczekiwane jako podpisane. Domyślam, się, że weryfikującym w obecnej konfiguracji jest Github, bo on by posiadał wszystkie klucze publiczne i jest w stanie zweryfikować dane commity. Moim zdaniem to dość słabe ogniwo na ten moment.

Reply:
W pełni się zgadzam. Ja na przykład podpisywałem swoje commity (jak musiałem) kluczem hardware'owym (Yubico) w taki sposób, że klucz prywatny nigdy nie dotknął żadnego twardego dysku ;). 


"Verified" przez GitHuba to jakby w ogóle nie było verified.

Reply:
@potiuk nice! Yubikey jest całkiem fajny. Ja używam Trezora którego bardzo polecam z hardware'owych kluczy.

Reply:
Ja mam 4 yubikeye. 2 na USB-a 2 na USB-C. Jak zaczynałem przygody z ProteGO to uratowałem dzięki nim moje konto na GitHubie bo miałem bardzo podejrzany phishing attack który dostałen na telefonie i nie widziałęm że domena jest "githubs.com" (w telefonie nie widać za bardzo). Tylko dlatego że miałem Yubikeye a nie SMS-y/TOTP nie wpadłem w pułapkę.